### DISCUSSION TITLE
GitHub desktop preventing Rstudio from opening<title>

### DISCUSSION THREAD

**Discussion by Hkloser** [community/community#44254] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am not sure if anyone else is having this problem but I have had this bug for many days now. I downloaded and set up GitHub desktop to work with Rstudio for global setup and the individual project I am working on.  It worked fine until I closed out of the apps and then tried to reopen them. For a bit, it seemed like it was working if I opened up Rstudio by opening the markdown directly from my computer files and as soon as GitHub desktop popped up to close out of that, but now that method is no longer working.  I couldn't get Rstudio to open at all because GitHub desktop was interfering. I uninstalled R and R studio and then re-downloaded them thinking it was them. That didn't solve the problem. I then uninstalled GitHub desktop and Rstudio was able to open and then re-downloaded GitHub desktop again. I thought the problem was solved but I then could not open Rstudio again for 2 days so now I have had to resolve to uninstall GitHub desktop every time I want to open Rstudio and then re-installing it to connect my work to the hub. This is all on my office desktop widows 64bit computer. 
I also have the same project set up on my mac laptop and it works fine. I don't have the desktop version of git installed, instead, it just opens in the pannel in Rstudio but I don't have any problems with that. 
Does anyone have any solutions other than just uninstalling and then downloading the app every time? Is this a bug other people are experiencing?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Hkloser ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
